nmrfx-processor/src/main/java/org/nmrfx/processor/datasets/Dataset.java

```diff
@@ -111,23 +112,25 @@ public Dataset(String fullName, String name, boolean writable, boolean useCacheF
             layout = headerIO.readHeader(raFile);
         }
         memoryMode = false;
-
-        DatasetParameterFile parFile = new DatasetParameterFile(this, layout);
-        parFile.readFile();
-        if (layout != null) {
-            createDataFile(raFile, writable);
-        }
+        //Only automatically set the freqDomains to true if they are all false
+        boolean noFreqDomains = IntStream.range(0, freqDomain.length).allMatch(i -> !freqDomain[i]);
         // Datasets that were not generated in NMRFx don't
         // have freqDomain  attribute set so set freq domain
         // here
-        boolean isFID =  (getNFreqDims() == 0) && ((!getFreqDomain(0) && (getComplex(0))));
-        if (!isFID && !getFreqDomain(0)) {
+        boolean isFID =  (getNFreqDims() == 0) && (!getFreqDomain(0) && (getComplex(0)));
+        if (!isFID && noFreqDomains) {
             int nFreq = getNFreqDims() == 0 ? nDim : getNFreqDims();
             for (int i = 0; i < nFreq; i++) {
                 setFreqDomain(i, true);
             }
         }
 
+        DatasetParameterFile parFile = new DatasetParameterFile(this, layout);
+        parFile.readFile();
+        if (layout != null) {
+            createDataFile(raFile, writable);
+        }
+
```

Comment:
The older NMRView files don't have freq domain set, but there wasn't a way to tell if the file was an old nv file, or if the file had just been processed without having any fourier transforms, so switched the ordering of when the parameter file was read, otherwise setting to true by default will overwrite the fdomain value in the par file.